Mapper/Errors.swift

```diff
@@ -5,5 +5,6 @@
  Custom implementations of Mappable, Convertible, or transformation functions can throw any error they desire
  */
 public struct MapperError: ErrorType {
+    @warn_unused_result
```

Comment:
Hmmm. Looks like this warning actually happens without `@warn_unused_result` as well. With slightly different text. Lets leave this here anyways to be explicit though.